Sources/SwiftUIPager/Pager+Buildable.swift

```diff
@@ -30,6 +30,11 @@ extension Pager: Buildable {
             .mutating(keyPath: \.contentLoadingPolicy, value: .eager)
     }
 
+    /// User can only swipe forward so in one direction
```

Comment:
can you also add a `- Parameter` and description please? check out any other modifier

Reply:
done